### PR TITLE
AFC-1103 - translates authorize to auktorisera

### PR DESCRIPTION
**Pull request by afeli76**
Updates res_sni, res_ssyk and partner_daily_notes

### CODE REVIEW

File: res_ssyk/i18n/sv_SE.po [hunk 1]
```diff
@@ -76,12 +76,12 @@ msgstr "Namn"
...
 #: model:ir.model.fields,field_description:res_ssyk.field_res_ssyk__code
 msgid "Official Code"
-msgstr "Officiell Kod"
+msgstr "Officiell kod"
 
 #. module: res_ssyk
 #: model:ir.model.fields,help:res_ssyk.field_res_ssyk__code
 msgid "Official code, group, sub-group or detail group."
-msgstr "Officiell kod, grupp, undergrupp eller detaljerad grupp."
+msgstr "Officiel kod, grupp, undergrupp eller detaljerad grupp."
```
Comment (FredrikArvas) [inline res_ssyk/i18n/sv_SE.po]: Officiell

File: partner_view_360/i18n/sv_SE.po [hunk 2]
```diff
@@ -371,28 +410,42 @@ msgid "Local office"
...
+#: model_terms:ir.ui.view,arch_db:partner_view_360.bankid
 #: model_terms:ir.ui.view,arch_db:partner_view_360.view_jobseeker_form
 #: model_terms:ir.ui.view,arch_db:partner_view_360.view_partner_employer_form
 msgid "Logo"
 msgstr "Logotyp"
 
+#. module: partner_view_360
+#: selection:res.partner,type:0
+msgid "Mailing Address"
+msgstr "E-postadress"
```
Comment (FredrikArvas) [inline partner_view_360/i18n/sv_SE.po]: Postadress

File: partner_view_360/i18n/sv_SE.po [hunk 3]
```diff
@@ -419,6 +472,12 @@ msgstr "Mina kontakter"
 msgid "Name"
 msgstr "Namn"
 
+#. module: partner_view_360
+#: model:ir.model.fields,field_description:partner_view_360.field_res_partner__name_com_reg_num
+#: model:ir.model.fields,field_description:partner_view_360.field_res_users__name_com_reg_num
+msgid "Name Com Reg Num"
+msgstr "Namn Com Reg Num"
```
Comment (MickeHolm) [inline partner_view_360/i18n/sv_SE.po]: What's this?

File: partner_view_360/i18n/sv_SE.po [hunk 4]
```diff
@@ -464,17 +521,17 @@ msgstr "Annan adress"
...
 #: selection:res.partner,registered_through:0
 msgid "PDM"
-msgstr ""
+msgstr "PDM"
 
 #. module: partner_view_360
 #: model_terms:ir.ui.view,arch_db:partner_view_360.view_jobseeker_form
 msgid "Personal Information"
-msgstr "Personlig information"
+msgstr "Personliguppgifter"
```
Comment (MickeHolm) [inline partner_view_360/i18n/sv_SE.po]: Personliga uppgifter

**Review by MickeHolm — CHANGES_REQUESTED** [review]

**Review by MickeHolm — APPROVED** [review]

**Pull request merged**